humble/distribution.yaml

```
@@ -4766,6 +4766,16 @@ repositories:
       url: https://github.com/ros-planning/navigation_msgs.git
       version: humble
     status: maintained
+  nicla_vision_ros2:
+    doc:
+      type: git
+      url: https://github.com/ADVRHumanoids/nicla_vision_ros2
```

These all need to end in `.git` to pass CI:

```suggestion
      url: https://github.com/ADVRHumanoids/nicla_vision_ros2.git
```

---

ok, sorry, I added it